### DISCUSSION TITLE
Support Asset Class, Asset Sub Class, and Tags in imported activity?

### DISCUSSION THREAD

**Discussion by sameleff** [ghostfolio/ghostfolio#2607] [Ideas]
Is there a way to add this information when importing activity via the csv/json importer or via the API? Or can it only be added item-by-item via the GUI once the data has been imported?
Asset & Asset Sub Classes appear to be only relevant to `valuable` type activity; but tags are applicable to all activity types.

**Comment by dtslvr** [comment]
Asset (sub) classes and tags are not supported via import API, yet. Are you interested in contributing these features?

**Reply by sameleff** [reply]
I'm certainly interested, but my familiarity with programming an API is limited to non-existent. Personally, and especially as a new user, I think one of the main areas I could add some assistance is via documentation for using the tool, both in the GUI setting as well as using the various importing methods (csv/json and API documentation). I realize the self-hosted areas of this project intentionally receive less documentation & support (and I have no objections with that whatsoever), but I think the lack of documentation currently poses a significant barrier to obtaining new users.
